styles/style.css

```diff
@@ -0,0 +1,1049 @@
+/*=======================================
+    GENERAL STYLES
+==================================================*/
+body {
+    overflow-x: hidden;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-family: 'Open Sans', sans-serif;
+    background-color: #272727;
+}
+
+.container {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 80px;
+}
+
+#top-container {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 330px;
+}
+
+h2 {
+    margin-top: 44px;
+    margin-bottom: 44px;
+    font-weight: bold;
+    text-transform: uppercase;
+    font-size: 16px;
+    letter-spacing: 8px;
+}
+
+p {
+    font-size: 16px;
+    line-height: 28px;
+    margin-bottom: 16px;
+    margin-top: 0px;
+    color: #ebebeb;
+}
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Raleway', sans-serif;
+}
+
+@media screen and (max-width: 750px) {

```

Comment:
I think it is better practise to have only one `@media` for one breakpoint. Now you have few media queries in few places in whole file.
What about having one media query `@media screen and (max-width: 750px)` on the bottom of file and inside this media query you can keep only modifications of global styles from top. Under this media query you could place another media query and place inside all styles modifications for that specyfic break point